packages/mds-daily/types.ts

```diff
@@ -18,11 +18,12 @@ export interface AgencyApiRequest extends express.Request {
   }
 }
 
+// TODO Extract shared types from mds-agency
 export interface TripsStats {
   single: number
-  singles?: any // FIXME
-  mysteries?: any // FIXME
-  mystery_examples?: any // FIXME
+  singles?: any // TODO
```

Comment:
This type is duplicated in `mds-agency`, we should move it to the shared `mds.d.ts` file.

Reply:
Michael just pushed a change that did this.